src/main/java/com/pedjak/gradle/plugins/dockerizedtest/ForkingTestClassProcessor.java

```diff
@@ -47,7 +47,7 @@ public void startProcessing(TestResultProcessor resultProcessor) {
     public void processTestClass(TestClassRunInfo testClass) {
         int i = 0;
         RuntimeException exception = null;
-        while (remoteProcessor == null && i < 10)
+        while (remoteProcessor == null && i < 20)
```

Comment:
did increasing to 20 really help?

Reply:
Yes, and i'd really make it retry for even longer - there is no point in stop retrying so fast given we know that the failures in 99% cases are caused by lack of the resource and it will eventually succeed. Retrying the whole test round is much more expensive and will not be that reliable.